### PR TITLE
Fixed incorrect optimization level default in vdb_ax

### PR DESCRIPTION
**Pull request by Idclip**

### CODE REVIEW

File: cmake/FindBlosc.cmake [hunk 1]
```diff
@@ -247,7 +249,9 @@ endforeach()
 # Reset library suffix
 
 set(CMAKE_FIND_LIBRARY_SUFFIXES ${_BLOSC_ORIG_CMAKE_FIND_LIBRARY_SUFFIXES})
+set(CMAKE_FIND_LIBRARY_PREFIXES ${_BLOSC_ORIG_CMAKE_FIND_LIBRARY_PREFIXES})
 unset(_BLOSC_ORIG_CMAKE_FIND_LIBRARY_SUFFIXES)
+unset(_BLOSC_ORIG_CMAKE_FIND_LIBRARY_PREFIXES)
```
Comment (richhones) [inline cmake/FindBlosc.cmake:255]: What are these changes related to? Do they need a comment in the pending changes?

Comment (Idclip) [reply cmake/FindBlosc.cmake:255]: Minor changes to support blosc lib changes in vcpkg

**Review by richhones — APPROVED** [review]
